builders/get-started/eth-compare/tx-fees.md

```diff
@@ -76,7 +120,7 @@ contract SayHello {
 }
 ```
 
-You can deploy this contract on both Moonbeam's TestNet, Moonbase Alpha, and Ethereum's TestNet, Sepolia, or feel free to access the contracts that have already been deployed to each network:
+You can deploy this contract on both Moonriver and Ethereum, or on Moonbeam's TestNet, Moonbase Alpha, and Ethereum's TestNet, Sepolia. The above contract has already been deployed to Moonbase Alpha and Sepolia. You can feel free to access these contracts at the following addresses:
 
 === "Moonbase Alpha"
```

Comment:
Please verify the contract on Moonbase Alpha 
Sepolia too if it's not too much trouble